root/user/UserEdits.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import EditList from '../edit/components/EditList';
+import EditorLink from '../static/scripts/common/components/EditorLink';
+
+type Props = {
+  +$c: CatalystContextT,
+  +editCountLimit: number,
+  +edits: $ReadOnlyArray<{...EditT, +id: number}>,
+  +pager: PagerT,
+  +refineUrlArgs?: {[argument: string]: string},
```

Comment:
```suggestion
  +edits: $ReadOnlyArray<$ReadOnly<{...EditT, +id: number}>>,
  +pager: PagerT,
  +refineUrlArgs?: {+[argument: string]: string},
```

Reply:
The `refineUrlArgs` change is probably not allowed (related to my comment above).